website/docs/dev_guide/high_level/converters.md

```diff
@@ -52,7 +52,7 @@ var config = new DynamoDbContextConfig(regionEndpoint, awsCredentials)
 };
 ```
 
-If converter can't be instantiated in advance and depends on the target value type, a custom converter factory can be implemented by inheriting from the `DdbConverterFactory` class and registering it for context the same way as other custom converters.
+If converter can't be instantiated in advance and depend on the target value type, a custom converter factory can be implemented by inheriting from the `DdbConverterFactory` class and registering it with the context the same way as other custom converters.
```

Comment:
Any specific reason why do you use `depend` here? For me, it sounds like `it depends`.

Reply:
That appears to be a typo on my part. I think the original "depends" is fine, but it needs to start with "If *the* converter"